scripts/globals/trust.lua

```diff
@@ -86,3 +115,43 @@ tpz.trust.spawn = function(caster, spell)
 
     return 0
 end
+
+tpz.trust.message = function(mob, id)
+    local master = mob:getMaster()
+    local offset = (mob:getTrustID() - 896) * 100
+    local party = master:getParty()
+
+    for _, member in ipairs(party) do
+        member:messageCombat(mob, offset + id, 0, 711)
```

Comment:
if its a per zone thing (as in, its in the mstring dats of each zone, even if they all have teh same ID now) it should really be in the IDs.lua anyway just for thise reason. if its not in the zones dat, we generaly won't need to worry about it shifting.

I most zones we have several IDs that are generally the same in every zone - we still IDs.lua them because they will eventually get shifted,
![image](https://user-images.githubusercontent.com/6871475/87230484-5e15bd00-c37e-11ea-9695-ff97cd618c2d.png)